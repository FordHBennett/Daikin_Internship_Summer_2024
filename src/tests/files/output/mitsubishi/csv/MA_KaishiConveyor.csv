tag_name,address
sdf,R<Int16>1
sdf,R<Int16>0
sdf,R<Int16[13]>50
sdf,R<Int16[14]>50
sdf,R<Int16[15]>50
,R<Int16>1
,R<Int16>0
,R<Int16[13]>50
,R<Int16[14]>50
,R<Int16[15]>50
