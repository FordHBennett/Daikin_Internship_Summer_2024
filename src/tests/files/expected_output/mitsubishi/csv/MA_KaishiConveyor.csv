tag_name,address
PHS_PLC_Heartbeat,R<Int16>1
PLC_PHS_Heartbeat,R<Int16>0
KaishiCVA_Running,R<Int16>50.13
KaishiCVB_Running,R<Int16>50.14
KaishiCVC_Running,R<Int16>50.15
